jbpm-test-util/src/main/java/org/jbpm/test/persistence/scripts/TestPersistenceContextBase.java

```diff
@@ -141,7 +141,7 @@ public void executeScripts(final File scriptsRootFolder, ScriptFilter scriptFilt
     public void executeScripts(final File scriptsRootFolder, ScriptFilter scriptFilter,
                                DataSource dataSource, String defaultSchema) throws IOException, SQLException {
         final File[] sqlScripts = TestsUtil.getDDLScriptFilesByDatabaseType(scriptsRootFolder, databaseType, scriptFilter);
-        if (sqlScripts.length == 0 && scriptFilter.hasOption(Option.DISALLOW_EMTPY_RESULTS)) {
+        if (sqlScripts.length == 0 && scriptFilter.hasOption(Option.DISALLOW_EMPTY_RESULTS) && scriptFilter.isSupportedDatabase(databaseType)) {
```

Comment:
Is really needed this last part of the comparison?

Reply:
No, not really - removed.